src/dbos-runtime/config.ts

```diff
@@ -262,7 +262,7 @@ export function constructPoolConfig(configFile: ConfigFile, cliOptions?: ParseOp
 
     databaseName = configFile.database.app_db_name;
     // Construct the database name from the application name, if needed
-    if (databaseName === undefined) {
+    if (databaseName === undefined || databaseName === '') {
```

Comment:
This is much more intuitive and handles `app_db_name` set to an empty env var: falls back on the app name.